measure/sbin/griffin-tool.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
+CONFDIR="${BASEDIR}/conf"
+
+. "${BASEDIR}/bin/griffin-env.sh"
+
+if [ ! $# -ge 2 ]; then
+  echo "env file and dq file must be provided!"
+  exit 1
+fi
+
+envFile=$1
+if [ ! -f ${envFile} ];then
+  envFile="${CONFDIR}/${envFile}"
```

Comment:
we shouldn't assume that the configs files will be available in the config directory. If the configs do not exist, we can fail the process directly.